src/scripts/header/mobile-menu.ts

```
@@ -0,0 +1,158 @@
+
+/**
+ * Gestionnaire du menu mobile
+ * Gère l'ouverture/fermeture et les animations du menu mobile
+ * @class MobileMenu
+ */
+export class MobileMenu {
+  private toggleButton: HTMLButtonElement | null;
+  private overlay: HTMLElement | null;
+  private content: HTMLElement | null;
+  private navLinks: NodeListOf<HTMLElement>;
+  private isOpen: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)